deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -103,8 +103,10 @@ spec:
     * OpenShift Pipelines
     ## Installation
     CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in default `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-workspaces` namespace.
-    ##### Prerequisite:
-    * To finish the installation of Red Hat CodeReady Workspaces Operator, it is necessary that at least one real user exists in the OpenShift cluster.
+    ### Installation Pre-requisites
+    * OAuth-based authentication has been enabled and configured for the OpenShift cluster.
+    * One regular user (not the default kubeadmin) exists in the OpenShift cluster.
+    You can find information about configuring the cluster OAuth in [OpenShift Container Platform documentation](https://docs.openshift.com/container-platform/4.2/authentication/understanding-identity-provider.html).
```

Comment:
I've slightly modified it - we need one empty line so the last sentence appears on the new line, otherwise, it's appended to the last bullet. 9782d82

Reply:
to do that I had to slightly modify the generation script: https://github.com/codeready-toolchain/api/pull/79